src/Extension/CoreExtension.php

```diff
@@ -1216,41 +1217,42 @@ function twig_test_empty($value)
  *
  * @return bool true if the value is traversable
  */
-function twig_test_iterable($value)
+function twig_test_iterable($value): bool
 {
     return $value instanceof \Traversable || \is_array($value);
 }
 
-/**
- * Renders a template.
- *
- * @param array        $context
- * @param string|array $template      The template to render or an array of templates to try consecutively
- * @param array        $variables     The variables to pass to the template
- * @param bool         $withContext
- * @param bool         $ignoreMissing Whether to ignore missing templates or not
- * @param bool         $sandboxed     Whether to sandbox the template or not
- *
- * @return string The rendered template
- */
-function twig_include(Environment $env, $context, $template, $variables = [], $withContext = true, $ignoreMissing = false, $sandboxed = false)
+    /**
+     * Renders a template.
+     *
+     * @param array $context
+     * @param string|array $template The template to render or an array of templates to try consecutively
+     * @param array $variables The variables to pass to the template
+     * @param bool $withContext
+     * @param bool $ignoreMissing Whether to ignore missing templates or not
+     * @param bool $sandboxed Whether to sandbox the template or not
+     *
+     * @return string The rendered template
+     * @throws LoaderError|SyntaxError
+     */
+function twig_include(Environment $env, array $context, $template, array $variables = [], bool $withContext = true, bool $ignoreMissing, bool $sandboxed): string
 {
-    $alreadySandboxed = false;
```

Comment:
The removal of this `$alreadySandboxed` changes the behavior of the method. Why ?